main.tf

```diff
@@ -1,13 +1,28 @@
-resource "random_integer" "example" {
+# -----------------------------------------------------------------------------------------------------------------------
+# Subscribe the Acccount to Security Hub
+# -----------------------------------------------------------------------------------------------------------------------
+resource "aws_securityhub_account" "enable_us_east_1" {
   count = module.this.enabled ? 1 : 0
+}
+
+# -----------------------------------------------------------------------------------------------------------------------
+# Optionally subscribe to Security Hub Standards 
+# https://docs.aws.amazon.com/securityhub/latest/userguide/securityhub-standards.html
+# -----------------------------------------------------------------------------------------------------------------------
+resource "aws_securityhub_standards_subscription" "enable_cis_1_2_0" {
```

Comment:
as @osterman mentioned, we probably can use a variable like 

```
variable "rulesets" {
  default = []
}
```
with the values like 

```
[ "ruleset/cis-aws-foundations-benchmark/v/1.2.0", 
"standards/aws-foundational-security-best-practices/v/1.0.0",
"standards/pci-dss/v/3.2.1" ]
```

(and others if necessary)

and we `for_each` over the values

Reply:
in this case, the module interface looks cleaner, and we can add more rules later